### PR TITLE
10-10CG Schema Change: Removing Gender

### PR DESCRIPTION
**Pull request by kevinmirc**
Fixes: https://github.com/department-of-veterans-affairs/va.gov-team/issues/13098

**Description**:
The 10-10CG will be removing the option to select gender as "Unknown" and gender will no longer be required. This PR supports those schema changes.

**Related PRs**
Schema: https://github.com/department-of-veterans-affairs/vets-json-schema/pull/502
UI: https://github.com/department-of-veterans-affairs/vets-website/pull/13993

### CODE REVIEW

File: spec/services/form1010cg/service_spec.rb [hunk 1]
```diff
@@ -14,7 +14,6 @@
         },
         'ssnOrTin' => Faker::IDNumber.valid.remove('-'),
         'dateOfBirth' => Faker::Date.between(from: 100.years.ago, to: 18.years.ago).to_s,
-        'gender' => %w[M F].sample,
```
Comment (erluti) [inline spec/services/form1010cg/service_spec.rb:17]: so this should have no gender key by default?

**Review by erluti — APPROVED** [review]
LGTM, minor question

File: spec/services/form1010cg/service_spec.rb (hunk 1 above)
Comment (LinnJS) [reply spec/services/form1010cg/service_spec.rb:17]: I think this is true since gender is no longer required

Comment (kevinmirc) [reply spec/services/form1010cg/service_spec.rb:17]: Yes, gender should not be present in the minimalist example.

**Pull request merged**